### PR TITLE
Add a new gulp build task

### LINKED ISSUE #13: Add a new `gulp build` task to run the entire build without a watcher

### PR DESCRIPTION
**Pull request by anita-chouhan**
Fixes #13

### CODE REVIEW

File: assets/package.json [hunk 1]
```diff
@@ -4,8 +4,12 @@
   "version": "0.0.1",
   "license": "UNLICENSED",
   "private": true,
-  "main": "gulp.js",
+  "main": "assets/gulp.js",
   "author": "Miranj",
+  "scripts": {
+    "dev": "cd assets && gulp",
```
Comment (rungta) [inline assets/package.json:10]: This assumes the `package.json` file is in the root folder, not in the `assets` folder. But right now on this branch the package file is still in the assets folder, so these commands will fail, right?

File: assets/package.json [hunk 2]
```diff
@@ -4,8 +4,12 @@
   "version": "0.0.1",
   "license": "UNLICENSED",
   "private": true,
-  "main": "gulp.js",
+  "main": "assets/gulp.js",
```
Comment (rungta) [inline assets/package.json:7]: Right now `package.json` and `gulpfile.js` are in the same folder itself, right?

**Review by rungta — CHANGES_REQUESTED** [review]

File: assets/package.json (hunk 2 above)
Comment (anita-chouhan) [reply assets/package.json]: Reverted.

File: assets/package.json (hunk 1 above)
Comment (anita-chouhan) [reply assets/package.json]: This needs to be done after moving the `package.json` to _root_ in a different branch. I have removed the modification.

**Comment by anita-chouhan** [comment]
I think need to merge the _move package.json to the root folder_ in this branch first and then update the instructions again in the  _Readme_ file.

**Review by rungta — APPROVED** [review]

**Pull request merged**